src/esn.calendar.libs/app/components/event/form/event-form.controller.spec.js

```diff
@@ -1941,6 +1941,49 @@ describe('The CalEventFormController controller', function() {
       });
     });
 
+    describe('the isValidURL method', function() {
+      it('should return true if the location is valid', function() {
+
+        scope.event = CalendarShell.fromIncompleteShell({
+          title: 'title',
+          path: '/calendars/' + owner._id + '/' + calendarTest.id + '/eventID',
+          start: moment('2016-12-08 12:30'),
+          end: moment('2016-12-08 13:30'),
+          location: 'http://123.com',
+          etag: '123456',
+          alarm: {
+            trigger: '-PT1M',
+            attendee: 'test@open-paas.org'
+          }
+        });
+
+        initController();
+
+        expect(scope.isValidURL(scope.event.location)).to.be.true;
```

Comment:
```suggestion
        expect(scope.isValidURL(scope.editedEvent.location)).to.be.true;
```
otherwise the call to `initController()` won't make sense since we assigned the value ourselves [here](https://github.com/OpenPaaS-Suite/esn-frontend-calendar/pull/342/files#diff-ff6f01e59044d35241c8bf100db2a8e04257c139580a479de6831373d459163fR1947)